runtime/doc/api.txt

```diff
@@ -3108,6 +3108,195 @@ nvim_tabpage_set_var({tabpage}, {name}, {value})
+                              patterns to match against. See
+                              |autocmd-pattern|.
+                            • buffer: (integer) optional, buffer number for
+                              buffer local autocmds (can't be used together
+                              with pattern). See |autocmd-buflocal|.
+                            • desc: (string) optional, description of the
+                              autocmd.
+                            • callback: (function|string) optional, Lua
+                              function or Vim function (as string) to execute
+                              on event.
+                            • command (string) optional, vimscript command to
+                              execute on event (can't be used together with
+                              callback). Eg. opts.command = "let g:value_set =
+                              v:true"
+                            • once (boolean) optional, defaults to false. Run
+                              the autocmd only once. See |autocmd-once|.
+                            • nested: (boolean) If true, execute nested
+                              autocmds |autocmd-nested|.
+
+                See also: ~
+                    autocmd
+                    autocmd-groups
+                    autocmd-events
+                    autocmd-pattern
+                    autocmd-buflocal
+                    autocmd-once
+                    autocmd-nested
+
+nvim_del_augroup_by_id({id})                        *nvim_del_augroup_by_id()*
+                Delete an autocommand group by id.
+
+                NOTE: behavior differs from |augroup-delete|. When deleting an
+                augroup, autocmds contained by this augroup will also be
+                deleted and cleared. This augroup will no longer exist
+
+                Parameters: ~
+                    {id}  Integer The id returned by nvim_create_augroup.
+
+                See also: ~
+                    nvim_del_augroup_by_name
+                    nvim_create_augroup
+
+nvim_del_augroup_by_name({name})                  *nvim_del_augroup_by_name()*
+                Delete an autocommand group by name.
+
+                NOTE: behavior differs from |augroup-delete|. When deleting an
+                augroup, autocmds contained by this augroup will also be
+                deleted and cleared. This augroup will no longer exist.
+
+                See also: ~
+                    autocommand-groups
+
+nvim_del_autocmd({id})                                    *nvim_del_autocmd()*
+                Delete an autocommand by id.
+
+                Parameters: ~
+                    {id}  Integer The id returned by nvim_create_autocmd
+
+                See also: ~
+                    nvim_create_augroup
+                    nvim_del_augroup_by_name
+
+nvim_do_autocmd({*opts})                                   *nvim_do_autocmd()*
```

Comment:
What does `*` mean here? That `{opts}` can be omitted? What happens then?

Reply:
I think this means it is a pointer.